pages/documentation/simulation/flowdecomposition/index.md

```diff
@@ -40,11 +38,10 @@ Below is the concrete description of the algorithm implemented in PowSyBl.
 
 ### Net positions computation
 
-Countries' net position computation is done using AC loadflow in the initial network, before any other alteration of the input.
+Countries' net position computation is done once for all on basecase using AC loadflow in the initial network, before any other alteration of the input.
 
-The net position of a country is calculated as the sum of :
-- The injection of all dangling lines connected in the country
-- The mean leaving flow of all AC  and HVDC line interconnections (losses are shared equally between both countries)
+The net position of a country is calculated as the sum of the mean leaving flow of all AC and HVDC line interconnections
```

Comment:
What object/method are you using for leaving flow calculation? Because we have now `Boundary` object that could avoid making a mean. And how do you do with HVDC links?

Reply:
I was not aware of the new Boundary object, so basically i do not use it (the implementation is simply done by getting getTeminal1().get(P) and getTerminal2().getP()). I do not know the potential impact on CGMES, but to be "accurate", boundary or not, the result should be the same at least with UCTE files. As I am describing the algorithm here (and not its specific implementation) I think it is better to keep this description - even if we go to a refactoring using Boundary object.

For HVDC (I mean IIDM HVDC) the algorithm is exactly the same, as all we are using are methods from Branch interface
On the other hand, for "UCTE HVDC" that means HVDC that are not modelled, current implementation does not take into account the HVDC flow as an exchange (so basically here it is better to call it AC line exchange, but it is due to the fact that we do not have any way to automatically create HVDC objects from UCTE files without extra configuration).

For more detail on the implementation, it is all enclosed in one file https://github.com/powsybl/powsybl-entsoe/blob/main/flow-decomposition/src/main/java/com/powsybl/flow_decomposition/NetPositionComputer.java.